pkg/sql/logictest/testdata/logic_test/as_of

```diff
@@ -23,6 +23,15 @@ SELECT * FROM t AS OF SYSTEM TIME INTERVAL '-1ns'
 ----
 2
 
+# Verify that we can use computed expressions.
```

Comment:
Add another test with `AOST now()` to verify non-constant expressions don't work.